src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -965,38 +965,7 @@ public void RemoveOrganelle(Hex hex)
 
     public float CalculateSpeed()
     {
-        float microbeMass = Constants.MICROBE_BASE_MASS;
-
-        float organelleMovementForce = 0;
-
-        Vector3 forwardsDirection = new Vector3(0, 0, -1);
-
-        foreach (var organelle in editedMicrobeOrganelles.Organelles)
-        {
-            microbeMass += organelle.Definition.Mass;
-
-            if (organelle.Definition.HasComponentFactory<MovementComponentFactory>())
-            {
-                Vector3 organelleDirection = (Hex.AxialToCartesian(new Hex(0, 0))
-                    - Hex.AxialToCartesian(organelle.Position)).Normalized();
-
-                float directionFactor = organelleDirection.Dot(forwardsDirection);
```

Comment:
This is more correct math, but even this isn't 100% exact, but if you move back to this math, I'd consider it good enough (with a TODO comment linking https://github.com/Revolutionary-Games/Thrive/issues/2095)